docs/content-contribution/overall-process.md

```diff
@@ -8,7 +8,7 @@ The overall process for content contribution starts with you.
 
 1. You're reading documentation on [SAP Help Portal](http://help.sap.com). You come across something you're not sure is entirely accurate, or you've spotted a typo, or you think that there's a small addition or amendment that you can offer that would help clarify or otherwise add value to what's already there.
 
-2. You look for the **Edit** button in the function bar at the top of the page. If there is no **Edit** button, the documentation is not yet part of SAP's Open Documentation Initative.
+2. You look for the *Edit page* option via the *Feedback* link in the function bar near the top of the page. If there is no such option, the documentation is not yet part of SAP's Open Documentation Initative.
 
 ![Content contribution link](../assets/content-contribution-link.png)
```

Comment:
The indentation for the graphic does not look correct (three spaces are missing, see [preview](https://github.com/SAP-docs/contribution-guidelines/blob/ca12562f901417d5b37388d6eeb60ecf73e88371/docs/content-contribution/overall-process.md)):
<img width="737" alt="image" src="https://github.com/SAP-docs/contribution-guidelines/assets/34343894/3e943a79-0035-449d-9eaa-69d15a30bed6">

Reply:
This was incorrect before. In fact looking again at this section has prompted me to rework it slightly, removing the extra complexity of a numbered list.